lib/MCToIRLifter.cpp

```diff
@@ -285,8 +287,32 @@ void MCToIRLifter::VisitDelayedInstruction(const remill::Instruction &inst,
   }
 }
 
-void MCToIRLifter::VisitInstruction(remill::Instruction &inst,
-                                    llvm::BasicBlock *block) {
+/*
```

Comment:
Use `//` for comments instead of `/*`. Indent the example llvm code with respect to the rest of the comment, e.g.

```
// blah blahb lbha blah blah blah blahb
//
//         ir ir i r irir ir ir
//
// blah blah blah lbah
//
//         ir ir ir ir
//
// yeet.
```